src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -98,4 +110,9 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
     f"DataWord(0x$n%02x)" //would be even better to add a leading zero if odd number of digits
 
   def toBigInt: BigInt = n
+
+  /**
+    * @return Size in bytes excluding the leading 0 bytes
+    */
+  def byteSize: Int = if (isZero) 0 else (n.bitLength - 1) / 8 + 1
```

Comment:
maybe we can add
`1 + floor((n.bitLength - 1) / 8) = 1 + floor(log_256(n))`
as comment, because it was not obvious for me at first glance